tests/tests.py

```diff
@@ -31,6 +35,19 @@ def setUp(self):
 
         self.client.login(username='admin', password='password')
 
+    @contextmanager
+    def withAssertNumQueriesLessThen(self, value, using=DEFAULT_DB_ALIAS):
```

Comment:
I think there is a typo here and it should be named `withAssertNumQueriesLessThan`.